samples/ingestion/ingestion-client/FetchTranscription/TranscriptionProcessor.cs

```diff
@@ -145,7 +148,7 @@ private static async Task ProcessFailedTranscriptionAsync(string transcriptionLo
                 if (retryAudioFile && audio.RetryCount < FetchTranscriptionEnvironmentVariables.RetryLimit)
                 {
                     log.LogInformation($"Retrying transcription with name {fileName} - retry count: {audio.RetryCount}");
-                    var sbMessage = new ServiceBusMessage
+                    var sbMessage = new Connector.ServiceBusMessage
```

Comment:
Why is this being added?

Reply:
@JoshLove-msft, There are ServiceBusMessage class in both Connector and Azure.Messaging.ServiceBus.